sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAuthoringClient.cs

```diff
@@ -3186,70 +2516,60 @@ public virtual async Task<Operation<BinaryData>> ImportProjectAsync(WaitUntil wa
         /// <exception cref="ArgumentNullException"> <paramref name="projectName"/> or <paramref name="content"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="projectName"/> is an empty string, and was expected to be non-empty. </exception>
         /// <remarks>
-        /// Schema for <c>Request Body</c>:
+        /// Below is the JSON schema for the request and response payloads.
+        /// 
+        /// Request Body:
+        /// 
+        /// Schema for <c>ExportedProject</c>:
         /// <code>{
-        ///   projectFileVersion: string (required),
-        ///   stringIndexType: &quot;Utf16CodeUnit&quot; (required),
+        ///   projectFileVersion: string, # Required. The version of the exported file.
+        ///   stringIndexType: &quot;Utf16CodeUnit&quot;, # Required. Specifies the method used to interpret string offsets. For additional information see https://aka.ms/text-analytics-offsets.
         ///   metadata: {
-        ///     projectKind: &quot;Conversation&quot; | &quot;Orchestration&quot; (required),
+        ///     projectKind: &quot;Conversation&quot; | &quot;Orchestration&quot;, # Required. Represents the project kind.
         ///     settings: {
-        ///       confidenceThreshold: number (required)
-        ///     },
-        ///     projectName: string (required),
-        ///     multilingual: boolean,
-        ///     description: string,
-        ///     language: string (required)
-        ///   } (required),
+        ///       confidenceThreshold: number, # Required. The threshold of the intent with the highest confidence, at which the prediction will automatically be changed to &quot;None&quot;.
+        ///     }, # Optional. The project settings.
+        ///     projectName: string, # Required. The new project name.
+        ///     multilingual: boolean, # Optional. Whether the project would be used for multiple languages or not.
+        ///     description: string, # Optional. The project description.
+        ///     language: string, # Required. The project language. This is BCP-47 representation of a language. For example, use &quot;en&quot; for English, &quot;en-gb&quot; for English (UK), &quot;es&quot; for Spanish etc.
+        ///   }, # Required. Represents the project metadata.
         ///   assets: {
-        ///     projectKind: &quot;Conversation&quot; | &quot;Orchestration&quot; (required)
-        ///   }
+        ///     projectKind: &quot;Conversation&quot; | &quot;Orchestration&quot;, # Required. The type of project containing the assets.
```

Comment:
Based on @chunyu3's [comment](https://github.com/Azure/azure-sdk-for-net/pull/29404#discussion_r904447728), I don't think this will be a useful UX without **inner** polymorphic model representation or at least links to REST APIs. The inner vs. outer distinction was not communicated previously, or I'd have raised concerns last week.

@tg-msft @KrzysztofCwalina should I instead copy affected methods with docs to my hand-written class and add links? @chunyu3 @ArcturusZhang is there any simpler way to add to documentation - likes links for more info - without taking over otherwise-generated code?

/cc @mayurid @kristapratico @mshaban-msft @annelo-msft